Makefile

```diff
@@ -163,15 +163,17 @@ uninstall:
 
 dist: clean validate man dist/$(name)-$(distversion).tar.gz
 
+# most of the sed stuff should be skipped if $(distversion) == $(version)
+# except RELEASE_DATE= and perhaps the Version in $(dscfile)
 dist/$(name)-$(distversion).tar.gz:
 	@echo -e "\033[1m== Building archive $(name)-$(distversion) ==\033[0;0m"
 	rm -Rf build/$(name)-$(distversion)
 	mkdir -p dist build/$(name)-$(distversion)
 	tar -c --exclude-from=.gitignore --exclude=.gitignore --exclude=".??*" * | \
 		tar -C build/$(name)-$(distversion) -x
-	@echo -e "\033[1m== Rewriting $(specfile), $(dscfile) and $(rearbin) ==\033[0;0m"
+	@echo -e "\033[1m== Rewriting build/$(name)-$(distversion)/{$(specfile),$(dscfile),$(rearbin)} ==\033[0;0m"
 	sed -i.orig \
-		-e 's#^Source:.*#Source: https://sourceforge.net/projects/rear/files/rear/${version}/$(name)-${distversion}.tar.gz#' \
+		-e 's#^Source:.*#Source: $(name)-${distversion}.tar.gz#' \
```

Comment:
So this `sed` patches just the dist archive name into the `Source` field of the spec file, doesn't that make it completely irrelevant what is says in the actual spec file below, as every `make rpm` includes also a `make dist` before?

Reply:
I think it is good to document where the canonical source can be obtained from.